federation-js/src/service/printCoreSchema.ts

```diff
@@ -171,10 +184,14 @@ function printScalar(type: GraphQLScalarType, options?: Options): string {
 function printObject(type: GraphQLObjectType, options?: Options): string {
   const interfaces = type.getInterfaces();
   const implementedInterfaces = interfaces.length
-    ? ' implements ' + interfaces.map(i => i.name).join(' & ')
+    ? ' implements ' + interfaces.map((i) => i.name).join(' & ')
     : '';
 
-  // Federation change: print `extend` keyword on type extensions.
+  // TODO: I can't figure out why this is here. I wrote this 8 months ago. Nothing
+  // in the history explains the reason this is needed. When do we have an `extend type`
+  // in the final composed schema?
```

Comment:
I can't think of a reason for having `extend type` in the composed schema either.

Reply:
https://github.com/apollographql/federation/pull/588#discussion_r598878526

eadfe94